packages/block-library/src/quote/transforms.js

```diff
@@ -122,6 +122,14 @@ const transforms = {
 		{
 			type: 'block',
 			blocks: [ 'core/paragraph' ],
+			isMatch: ( { citation }, block ) => {
+				return (
+					! RichText.isEmpty( citation ) ||
+					block.innerBlocks.some(
+						( { name } ) => name === 'core/paragraph'
```

Comment:
I think this transform should still be possible. Unfortunately the meaning has changed from "convert to paragraph" to just unwrap the entire thing. If we change this, it will no longer be possible to unwrap the quote?

Reply:
See #51934

Reply:
> I think this transform should still be possible.

Possibly. How should it work ... convert any non-paragraphs blocks to paragraphs? We can try running transforms to paragraphs for any inner block types that have that transform. The `isMatch` would have to check if there are available transforms for all inner block types, but it might not be too bad.

The other option is to allow it to simply unwrap, but I think it'd be weird if it's called a Paragraph transform, and yet it doesn't actually convert to paragraphs.

> If we change this, it will no longer be possible to unwrap the quote?

There's still an 'Ungroup' option in the block settings menu that unwraps the quote. It's not surfaced in the transform menu. It seems to work for any inner block types that quote may have.

I only found it because it's mentioned in the e2e tests.

Reply:
For what it's worth, "unquote" makes logical sense to me as a permanent option.

I'm likening it to markdown.

To wrap content in a quote we add `>`

> There are **multiple** _entities_ [inside](http://localhost) this quote 😄 

Removing the `>` will just 'unwrap' it, regardless of what the internal entities are.